app/src/partner/java/cm/aptoide/pt/store/view/home/HomeFragment.java

```diff
@@ -1,484 +0,0 @@
-import cm.aptoide.pt.networking.image.ImageLoader;
-import cm.aptoide.pt.preferences.PartnersSecurePreferences;
-import cm.aptoide.pt.repository.RepositoryFactory;
-import cm.aptoide.pt.search.SearchNavigator;
-import cm.aptoide.pt.search.SuggestionCursorAdapter;
-import cm.aptoide.pt.search.analytics.SearchAnalytics;
-import cm.aptoide.pt.search.suggestions.TrendingManager;
-import cm.aptoide.pt.search.view.AppSearchSuggestionsView;
-import cm.aptoide.pt.search.view.SearchSuggestionsPresenter;
-import cm.aptoide.pt.store.StoreTheme;
-import cm.aptoide.pt.store.view.StoreFragment;
-import cm.aptoide.pt.store.view.StorePagerAdapter;
-import cm.aptoide.pt.updates.UpdateRepository;
-import cm.aptoide.pt.utils.AptoideUtils;
-import cm.aptoide.pt.view.BackButton;
-import cm.aptoide.pt.view.custom.BadgeView;
-import com.jakewharton.rxbinding.support.v7.widget.RxToolbar;
-import com.jakewharton.rxbinding.view.RxView;
-import com.trello.rxlifecycle.android.FragmentEvent;
-import java.text.NumberFormat;
-import javax.inject.Inject;
-import rx.Observable;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
-import rx.subjects.PublishSubject;
-
-/**
- * Created by neuro on 09-05-2016.
- */
-public class HomeFragment extends StoreFragment {
-
-  public static final String FACEBOOK_PACKAGE_NAME = "com.facebook.katana";
-
-  @Inject AnalyticsManager analyticsManager;
-  @Inject NavigationTracker navigationTracker;
-  private DrawerLayout drawerLayout;
-  private NavigationView navigationView;
-  private BadgeView updatesBadge;
-  private BadgeView notificationsBadge;
-  private UpdateRepository updateRepository;
-  private AptoideAccountManager accountManager;
-  private AccountNavigator accountNavigator;
-  private TabNavigator tabNavigator;
-  private TextView userEmail;
-  private TextView userUsername;
-  private ImageView userAvatarImage;
-  private DrawerAnalytics drawerAnalytics;
-  private BackButton.ClickHandler backClickHandler;
-  private String defaultThemeName;
-  private AppSearchSuggestionsView appSearchSuggestionsView;
-  private CrashReport crashReport;
-  private SearchNavigator searchNavigator;
-  private TrendingManager trendingManager;
-  private SearchAnalytics searchAnalytics;
-
-  public static HomeFragment newInstance(String storeName, StoreContext storeContext,
-      String storeTheme) {
-    Bundle args = new Bundle();
-    args.putString(BundleKeys.STORE_NAME.name(), storeName);
-    args.putSerializable(BundleKeys.STORE_CONTEXT.name(), storeContext);
-    args.putSerializable(BundleKeys.STORE_THEME.name(), storeTheme);
-    HomeFragment fragment = new HomeFragment();
-    fragment.setArguments(args);
-    return fragment;
-  }
-
-  /**
-   * @return {@link HomeFragment} instance with default store, store context and theme
-   */
-  public static HomeFragment newInstance(String defaultStore, String defaultTheme) {
-    return newInstance(defaultStore, StoreContext.home, defaultTheme);
```

Comment:
Not related to this ticket, but some StoreContexts are not being used

Reply:
Done, good catch.